### DISCUSSION TITLE
Arm and Android version Filter APKMirror

### DISCUSSION THREAD

**Discussion by VasilisPngs** [rumboalla/apkupdater#556] [Ideas]
Αnother good addition would be to check the Arm version of the installed application so that the updated version replaces the same Arm version (v8a) or (v7a). It could also check the Android version of the installed version so that it does not suggest updating the application with a smaller Android version or just a different one.

**Comment by henricrammer** [comment]
Absolutely, checking the ARM and Android version before updating an app is super helpful—similar to how using the correct version of [Hill Climb Racing 2 Mod APK](https://hillclimbapks.com/) ensures everything runs smoothly with all mods working perfectly.
